libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/internal/ExitStyleGenerator.kt

```diff
@@ -0,0 +1,34 @@
+package com.mapbox.navigation.ui.maneuver.internal
+
+import android.content.res.Resources
+import android.graphics.Bitmap
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.text.SpannableStringBuilder
+import android.text.Spanned
+import android.text.style.ImageSpan
+import android.widget.TextView
+import com.mapbox.navigation.ui.utils.internal.extensions.drawableWithHeight
+import com.mapbox.navigation.ui.utils.internal.extensions.getAsBitmap
+
+object ExitStyleGenerator {
+
+    @JvmStatic
```

Comment:
We don't need to generate the Jvm static method, this is only an internal class. Same everywhere above.